mbof/migrations/0002_event_title.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9 on 2016-07-06 14:17
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('mbof', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='event',
+            name='title',
+            field=models.CharField(default='Untitled', max_length=50),
```

Comment:
Are the only cases for which the title will be "Untitled" are those events that already exist in the DB?  Does the logic in the model or controller classes require titles on new (or edited) events?  It could/should.